core/Nonce.php

```diff
@@ -99,6 +99,22 @@ public static function verifyNonce($id, $cnonce, $expectedReferrerHost = null)
         return true;
     }
 
+
+    /**
+     * Returns if a nonce is valid md5.
+     *
+     * @param string $md5
+     * @return bool
+     */
+    public static function isValidMd5($md5)
```

Comment:
btw if this method is in the Nonce class then a better name would be `isValidNonceFormat` or so. So we don't need to rename the method when it's no longer an md5 etc.

Reply:
I don't use it anymore, so deleted